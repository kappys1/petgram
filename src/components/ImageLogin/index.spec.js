// Category.test.js
/* global describe, it, expect */
import React from 'react'
// import renderer from 'react-test-renderer'
import { render } from '@testing-library/react'
import { ImageLogin } from '.'

describe('ImageLogin', () => {
  it('snapshot', () => {
    const { asFragment } = render(<ImageLogin />)
    expect(asFragment()).toMatchSnapshot()
  })
})
