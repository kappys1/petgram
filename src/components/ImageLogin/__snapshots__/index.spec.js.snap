// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageLogin snapshot 1`] = `
<DocumentFragment>
  <img
    alt="img-login"
    class="sc-bdfBQB dsfeoR"
    src="[object Object]"
  />
</DocumentFragment>
`;
